plugins/kubernetes-backend/src/service/KubernetesProxy.ts

```diff
@@ -0,0 +1,286 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { KubeConfig, bufferFromFileOrString } from '@kubernetes/client-node';
+import { Logger } from 'winston';
+import fetch from 'node-fetch';
+import * as https from 'https';
+
+import type { Request } from 'express';
+
+import { ClusterDetails, KubernetesClustersSupplier } from '../types/types';
+
+const HEADER_CONTENT_TYPE: string = 'Content-Type';
+const APPLICATION_JSON: string = 'application/json';
+
+const HEADER_KUBERNETES_CLUSTERS: string = 'X-Kubernetes-Clusters';
+
+const ERROR_BAD_REQUEST: number = 400;
+const ERROR_NOT_FOUND: number = 404;
+const ERROR_INTERNAL_SERVER: number = 500;
+
+export interface KubernetesProxyServices {
+  kcs: KubernetesClustersSupplier;
+}
```

Comment:
looks like CI wants this type to be exported by the whole package to address the warning in the api-report.md for this package...

Reply:
Yeah you aren't allowed to leave type definitions incomplete from the consumer's point of view.

Reply:
Removed this interface